USER_MANUAL.md

```diff
@@ -336,6 +363,345 @@ _TODO_
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Column headers in the file must correspond to an [emission rate source(ERS)](#ers), [non-repairable emission rate source(NR_ERS)](#nr_ers), or their equivalent when provided in a more granular setting.
+
+#### &lt;ERS&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** The name of a column in the [emissions file](#emissions_file) that is used to parameterize the emissions characteristics of all the general repairable emissions. This parameter will be overwritten by the equivalent value, when provided at a more granular scale.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** The column headers are case sensitive.
+
+#### &lt;LPR&gt;
+
+**Data Type:** Float (Numeric)
+
+**Default input:** None
+
+**Description:**  A numeric scalar representing the Leak Production Rate (LPR), which denotes the emission production rate for repairable emissions. New repairable emissions are generated using a site-level empirical LPR. LPR is the probability that a new repairable emission will arise, each day, for each site.  The LPR encapsulates various factors contributing to emission occurrences, such as facility age, management practices, predictive maintenance, and random chance. By setting the LPR in the virtual world parameter file, a uniform LPR is applied across all facility types, production types, facility ages, etc., unless specified otherwise at a more detailed level. For an extended discussion on LPR, see Fox et al. (2021).
+
+**Notes on acquisition:** While the "true" LPR is elusive, it can be estimated by dividing the number of leaks found during an LDAR survey at a facility by the number of days that have passed since the previous LDAR survey at the same facility. If this is done for a large number of survey intervals at a large number of facilities, one should eventually converge on a representative estimate. When LDAR-Sim is used, operator-specific LPR values should be estimated if sufficient data exist to do so.
+
+**Notes of caution:**  Available techniques for estimating LPR make a number of problematic assumptions. Ultimately, we have relatively poor data on LPR and the relationship between LPR and NRd. Modeling results are extremely sensitive to LPR. Given that LPR is elusive, we strongly recommend that a broad range of LPR values is evaluated in LDAR-Sim before any decisions are made. For more information, refer to discussions in the main text and supplementary information of Fox et al. (2021).
+
+When the parameter [multi](#multi) is set to false, it will affect the emission production rate observed in the simulation. In this scenario, new emissions won't be generated if there's already an existing emission for the specified emission source.
```

Comment:
Specified emission source might be confusing since at this level we don't specify it